longbits/long_bits.go

```diff
@@ -160,6 +161,10 @@ func (v *Bits224) AsByteString() ByteString {
 	return ByteString(v[:])
 }
 
+func (v Bits224) Compare(other Bits224) int {
```

Comment:
Could you please add this method to other long bits representations?

Reply:
done